src/tm_devices/drivers/pi/scopes/tekscope/tekscope.py

```diff
@@ -73,7 +82,180 @@ class TekScopeChannel:
     """Defines impedance at channel port."""
 
 
+class InternalAFGChannel(BaseAFGSourceChannel):
+    """Internal AFG channel driver."""
+
+    def __init__(self, tekscope: "TekScope") -> None:
+        """Create an Internal AFG channel.
+
+        Args:
+            tekscope: A TekScope.
+        """
+        super().__init__(pi_device=tekscope, channel_name="AFG")
```

Comment:
Where is the license check handled? We don't really want to create an object if there is no license, do we?